std/experimental/allocator/gc_allocator.d

```diff
@@ -111,6 +111,13 @@ struct GCAllocator
         return ((n + 4095) / 4096) * 4096;
     }
 
+    package pure nothrow @trusted void[] allocateZeroed(size_t bytes) shared
```

Comment:
Which `package`? Are you planning to make those public in a future PR?

Reply:
I think it would make sense, but this PR still works if it's entirely under the hood.

Reply:
Looks like this has to be `const` due to #6432